### DISCUSSION TITLE
Eval install, default. so-test fails, no such container.

### DISCUSSION THREAD

**Discussion by jvines-bazar** [Security-Onion-Solutions/securityonion#3173] [Unsupported Versions]
I have recently installed an eval setup of security onion. I am going through the essentials videos and when I get to so-test, it fails. so-status shows all green. My install follows the essentials videos, so I am a not sure why this wouldn't be working. 

I'd appreciate a pointer or two to get me going.

[root@securityonion ~]# sudo so-test
Replay functionality not enabled!  Enabling Now....

Note that you will need internet access to download the appropriate components
=========================================================================
Starting tcpreplay...

This could take a while if another Salt job is running.
Run this command with --force to stop all Salt jobs before proceeding.
=========================================================================
[ERROR   ] Command 'docker pull --disable-content-trust=false docker.io/securityonion/so-tcpreplay:2.3.21' failed with return code: 1
[ERROR   ] stderr: No valid trust data for 2.3.21
[ERROR   ] retcode: 1
[ERROR   ] {'pid': 20941, 'retcode': 1, 'stdout': '', 'stderr': 'No valid trust data for 2.3.21'}
[ERROR   ] Command 'docker tag securityonion/so-tcpreplay:2.3.21 securityonion:5000/securityonion/so-tcpreplay:2.3.21' failed with return code: 1
[ERROR   ] stderr: Error response from daemon: No such image: securityonion/so-tcpreplay:2.3.21
[ERROR   ] retcode: 1
[ERROR   ] {'pid': 20965, 'retcode': 1, 'stdout': '', 'stderr': 'Error response from daemon: No such image: securityonion/so-tcpreplay:2.3.21'}
[ERROR   ] Command 'docker push securityonion:5000/securityonion/so-tcpreplay:2.3.21' failed with return code: 1
[ERROR   ] stdout: The push refers to repository [securityonion:5000/securityonion/so-tcpreplay]
[ERROR   ] stderr: An image does not exist locally with the tag: securityonion:5000/securityonion/so-tcpreplay
[ERROR   ] retcode: 1
[ERROR   ] {'pid': 20977, 'retcode': 1, 'stdout': 'The push refers to repository [securityonion:5000/securityonion/so-tcpreplay]', 'stderr': 'An image does not exist locally with the tag: securityonion:5000/securityonion/so-tcpreplay'}
[ERROR   ] Failed to pull securityonion:5000/securityonion/so-tcpreplay:2.3.21: Error 404: manifest for securityonion:5000/securityonion/so-tcpreplay:2.3.21 not found: manifest unknown: manifest unknown
local:
----------
          ID: so-tcpreplayimage
    Function: cmd.run
        Name: docker pull --disable-content-trust=false docker.io/securityonion/so-tcpreplay:2.3.21
      Result: False
     Comment: Command "docker pull --disable-content-trust=false docker.io/securityonion/so-tcpreplay:2.3.21" run
     Started: 00:01:54.102824
    Duration: 1379.466 ms
     Changes:
              ----------
              pid:
                  20941
              retcode:
                  1
              stderr:
                  No valid trust data for 2.3.21
              stdout:
----------
          ID: so-tcpreplaytag
    Function: cmd.run
        Name: docker tag securityonion/so-tcpreplay:2.3.21 securityonion:5000/securityonion/so-tcpreplay:2.3.21
      Result: False
     Comment: Command "docker tag securityonion/so-tcpreplay:2.3.21 securityonion:5000/securityonion/so-tcpreplay:2.3.21" run
     Started: 00:01:55.484042
    Duration: 237.846 ms
     Changes:
              ----------
              pid:
                  20965
              retcode:
                  1
              stderr:
                  Error response from daemon: No such image: securityonion/so-tcpreplay:2.3.21
              stdout:
----------
          ID: so-tcpreplaypush
    Function: cmd.run
        Name: docker push securityonion:5000/securityonion/so-tcpreplay:2.3.21
      Result: False
     Comment: Command "docker push securityonion:5000/securityonion/so-tcpreplay:2.3.21" run
     Started: 00:01:55.723141
    Duration: 237.68 ms
     Changes:
              ----------
              pid:
                  20977
              retcode:
                  1
              stderr:
                  An image does not exist locally with the tag: securityonion:5000/securityonion/so-tcpreplay
              stdout:
                  The push refers to repository [securityonion:5000/securityonion/so-tcpreplay]
----------
          ID: so-tcpreplay
    Function: docker_container.running
      Result: False
     Comment: Failed to pull securityonion:5000/securityonion/so-tcpreplay:2.3.21: Error 404: manifest for securityonion:5000/securityonion/so-tcpreplay:2.3.21 not found: manifest unknown: manifest unknown
     Started: 00:01:56.417577
    Duration: 345.316 ms
     Changes:

Summary for local
------------
Succeeded: 0 (changed=3)
Failed:    4
------------
Total states run:     4
Total run time:   2.200 s
Replay functionality enabled.  Replaying PCAPs Now....
Error: No such container:path: so-tcpreplay:/opt/samples
Error: No such container: so-tcpreplay

PCAP's have been replayed - it is normal to see some warnings.

**Comment by weslambert** [comment]
This should be addressed in 2.3.30: https://github.com/Security-Onion-Solutions/securityonion/issues/2463
